### PR TITLE
feat(cocod): add option --check-config

### PR DESCRIPTION
**Pull request by nritsche**
Don't really start, just load config and exit.

Needed for https://github.com/chime-experiment/ch_ansible/issues/82

### CODE REVIEW

File: coco/master.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ class Master:
     Loads and keeps the config and endpoints. Endpoints are called through this module.
     """
 
-    def __init__(self, conf, reset):
+    def __init__(self, conf, reset, check_config):
```
Comment (anjakefala) [inline coco/master.py:48]: Why did you make this argument mandatory?

File: coco/master.py [hunk 2]
```diff
@@ -138,15 +147,16 @@ def __del__(self):
...
         Join the worker process.
         """
-        logger.info("Joining worker process...")
-        try:
-            self.redis_sync.rpush("queue", "coco_shutdown")
-        except BaseException as e:
-            logger.error(
-                f"Failed sending shutdown command to worker (have to kill it): {type(e)}: {e}"
-            )
+        if not self.check_config:
```
Comment (anjakefala) [inline coco/master.py:150]: This may not even be necessary because it seems `_kill_worker` checks if the worker exists before it kills. =) But nice catch!

Comment (nritsche) [reply coco/master.py:150]: Yeah, but it would still write some log lines that don't make much sense in this mode.

**Comment by anjakefala** [comment]
You may need to push a dummy commit that we rebase afterwards. T_T GitHub had server issues.

**Pull request merged**